lib/validates_zipcode/formatter.rb

```diff
@@ -0,0 +1,50 @@
+module ValidatesZipcode
+  module Formatter
+    class << self
+      # we assume the zipcode is valid for that country
+      def format(zipcode, country_alpha2)
```

Comment:
A `def self.format` can be a nice idea for convenience. But I'd make it just instantiate the class and call the instance method `def format`.

Reply:
It is a matter of taste, but ok. I have changed it.